README.md

```diff
@@ -57,8 +57,7 @@ It assumes you have obtained terminal session recording file by either:
 
 ### Use standalone player bundle in your HTML page
 
-Download latest version of the player bundle from
-[releases page](https://github.com/asciinema/asciinema-player/releases). You
+Build the latest version of the player bundle by following the instructions at [Development](#Development) below. You
```

Comment:
There are many people who use asciinema player who are not frontend devs and therefore the prebuilt assets are very useful to them so they don't need the whole toolchain. I just forgot to attach those files to the latest release. I just fixed that, so let's keep the original text here.

I guess we could automate attachment of the standalone assets via GH actions (release and prelease workflows).

Reply:
All right, makes sense and that's I guess preferred. Let me remove that then.

Reply:
https://github.com/asciinema/asciinema-player/pull/182/commits/f00219e3f3b8f2dba0e680098ba704cbe98a71ca